docs/source/topics/ref_minimum-system-requirements.adoc

```diff
@@ -12,6 +12,9 @@
 * 9 GB of free memory
 * 35 GB of storage space
 
+{prod} is supported only on AMD64 and Intel 64 processor architectures.
+{prod} does not support the ARM-based M1 architecture.
```

Comment:
With dev branch we are trying to have M1 support, we even published a dev-preview binary to support it.

Reply:
Let's create a docs issue for that and I'll address it with another PR. :)